src/application/service_tester.go

```diff
@@ -88,11 +88,13 @@ func (s *ServiceTester) getDockerComposeConfig() (types.DockerCompose, error) {
 	if err != nil {
 		return result, err
 	}
-	dockerConfigs := types.DockerConfigs{}
-	serviceDockerConfig := appDockerConfigs[s.Role]
-	serviceDockerConfig.DependsOn = s.getDependencyContainerNames()
-	serviceDockerConfig.Command = s.ServiceConfig.Development.Scripts["test"]
-	dockerConfigs[s.Role] = serviceDockerConfig
+	dockerConfigs, err := s.Initializer.getServiceDockerConfig(s.Role, s.ServiceConfig, types.ServiceData{Location: s.Role})
+	if err != nil {
+		return result, err
+	}
+	serviceConfig := dockerConfigs[s.Role]
+	serviceConfig.DependsOn = s.getDependencyContainerNames()
+	serviceConfig.Command = s.ServiceConfig.Development.Scripts["test"]
```

Comment:
It doesn't look like these `serviceConfig` objects actually being used anywhere, beyond being assigned values here.

Reply:
The `serviceConfig` variable is only used to update the command and dependsOn. Its a value in the `dockerConfigs` map which is what is returned

Reply:
It doesn't seem like this is actually modifying the underlying `dockerConfigs` object. I had to reassign `serviceConfig` to `dockerConfigs[s.Role]` in order for this method to return the right objects